custom_components/places/osm_client.py

```
@@ -88,24 +100,43 @@ def details_url(
 
     def update_sensor_name(self, sensor_name: str) -> None:
-        """Update the cached sensor name used in log messages."""
+        """Update the cached sensor name used in log messages.
+
+        Args:
+            sensor_name (str):
+                Sensor name cached for subsequent log messages.
+        """
         self._sensor_name = sensor_name
 
     async def get_json(self, url: str, name: str, *, use_cache: bool = True) -> Any | None:
```

Ruff (0.16.1)

[warning] Too many return statements (9 > 6)

(PLR0911)